tests/js/server/aql/aql-queries-simple.js

```diff
@@ -1355,6 +1355,9 @@ function ahuacatlQuerySimpleTestSuite () {
       }
       q += `RETURN v${cnt - 1})[0]\n`;
       q += "RETURN i + x";
+      require('console').log(q);
+      const db = require('@arangodb').db;
+      db._explain(q);
```

Comment:
Unrelated change and does not belong in this PR.

Reply:
Removed it